_data/config.yml

```diff
@@ -1,3 +1,3 @@
-grpc_release_branch: v1.0.0
```

Comment:
btw, v1.0.0 is actually a tag, not a branch, but I guess it's just the naming that wrong.